libgraph/test/test.c

```diff
@@ -17,13 +17,30 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#include <sys/time.h>
-
 #include <libgraph.h>
 
 #include "cursor.h"
 #include "font.h"
-#include "logo.h"
+#include "logo8.h"
+#include "logo16.h"
+#include "logo32.h"
+
+
+/* Forces all scheduled tasks completion */
+static int test_trigger(graph_t *graph)
+{
+	while (graph_trigger(graph));
+	return graph_commit(graph);
+}
+
+
+/* Forces next scheduled task to run immediately after vsync */
+static int test_vtrigger(graph_t *graph)
+{
+	while (graph_trigger(graph));
+	while (!graph_vsync(graph));

```

Comment:
Alek, please don't leave orphaned semicolons in lines.

Reply:
It's not orphaned, it represents empty instruction. IMHO it is a bad practice to leave `;` after `for` and `while` loops in the same line, because it can lead to an error, when somebody will place something in the loop eventaully:

`while (condition);`

goes into

```
while (condition);
   some_action();
```

On the other hand never versions of gcc will generate a warning in above case.

If you prefer

`while (condition);`

over

```
while (condition)
   ;
```

I think it would be best if I update our coding convention with this rule.

Reply:
I vote in favor of having semicolon in the next line for the cases like these as it improves the readability greatly :+1:

Reply:
Marek, Alek, I know that we try to preserve democracy :), but please don't leave semicolon in empty line. I use such notation for almost 20 years.